pkg/storage/pvc.go

```diff
@@ -173,6 +173,43 @@ func (builder *PVCBuilder) Create() (*PVCBuilder, error) {
 	return builder, nil
 }
 
+// Delete removes PVC from cluster.
+func (builder *PVCBuilder) Delete() error {
+	if valid, err := builder.validate(); !valid {
+		glog.V(100).Infof("persistentVolumeClaim %s is invalid- %v", builder.Definition.Name, err)
```

Comment:
I'd suggest instead of "invalid-" to use "invalid:"

Reply:
updated